### ISSUE TITLE
🐛 Bug Report:  Unable to create new User

### ISSUE THREAD

**Issue by RicardoSiqueira01** [appwrite/appwrite#8212]
### 👟 Reproduction steps

Hi, I was following this tutorial: "https://www.youtube.com/watch?v=ZBCUegTZF7M&list=LL&index=20&t=847s&ab_channel=JavaScriptMastery" 

And I was trying to integrate my database with my app, it is a ReactNative Project. I was using this code in the file appwrite.js, my idea was to develop a function to create a user in the database:

```
import { Client, Account, ID } from 'react-native-appwrite';

export const config = {
    endpoint: "https://cloud.appwrite.io/v1",
    platform: "com.pucvitual.recomecar",
    projectId: "665619be002faa3e07ef",
    databaseId: "66561bdf003dc4b9dca2",
    userCollectionId: "66561c44002c41d228b5",
    produtoCollectionId: "66561c8b00357fa4457b",
    comprasCollectionId: "66561eeb0025396a28cb",
    storageId: "6656206000240bcf7db2",
}

// Init your React Native SDK
const client = new Client();

client
    .setEndpoint(config.endpoint) // Your Appwrite Endpoint
    .setProject(config.projectId) // Your project ID
    .setPlatform(config.platform) // Your application ID or bundle ID.


    const account = new Account(client);

// Registrar usuário
export const createUser = () => {
    account.create(ID.unique(), 'me@example.com', 'senha', 'Jane Doe')
    .then(function (response) {
        console.log(response);
    }, function (error) {
        console.log(error);
    });
}
```

Same used by the guy on the video at 01:42:00. When I pressed the button that called the function createUser, this error appeared:

`TypeError: 0, _appwrite.default is not a function (it is undefined), js engine: hermes `

I am a begineer developer and I'm having a hard time trying to find what's wrong. I speak portuguese, so if you guys see some diferent words and some mistakes, I'm sorry, I'm still learning.

### 👍 Expected behavior

I was expecting to see this in the console: 
![Captura de tela 2024-05-28 174145](https://github.com/appwrite/appwrite/assets/106103247/ad0797c7-c145-4d8f-957b-b69027d83d2b)

### 👎 Actual Behavior

It actually gave me this error: `TypeError: 0, _appwrite.default is not a function (it is undefined), js engine: hermes `
![Captura de tela 2024-05-28 174403](https://github.com/appwrite/appwrite/assets/106103247/cb41a22c-01ce-46e7-af99-12fe123874a3)

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Windows

### 🧱 Your Environment

I'm using React Native SDK 0.1.0 and dont think I have customized it.
 

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@RicardoSiqueira01, thanks for creating this issue! 🙏🏼 I've seen this before when something was wrong with the code that lead to a compile time error. Double check you:

1. have the Appwrite SDK installed (you also might want to update. We just released version 0.4.0 of the react native SDK)
2. don't name any file `appwrite` 
3. there aren't any errors in your code

If you're still having problems, the best place to get help is to create a support post on our [Discord](https://appwrite.io/discord) as it would be better to troubleshoot there rather than on GitHub.

**Comment by RicardoSiqueira01** [comment]
Oh hi! Thanks for the support! I verified what was wrong and I found out the problem! Actually I made a mistake when I was importing the function, I didn't write the "{ }" when I was importing "createUser". I was testing the code and I started to erase some parts in the file appwrite.js, to see if the error would be reported in the console when the button was pressed. But I noticed that even erasing some parts, the same error continued appearing, so I thought, maybe the error is in the page that was calling it. I read my code and the code from the guy of the tutorial and I saw where it was, I will put a screenshot. In the end, I finally registered my first test-user and saw it in my Auth storage. Sorry to bother you guys creating this issue, It was me being a goofball! But something that I didn't understand was why the console reported the error being in the "_appwrite.default" and not in the page that was importing the function, maybe I missed something. But thanks again and sorry for the issue, have nice day or night!

![Error](https://github.com/appwrite/appwrite/assets/106103247/41ded65e-fb6d-4c5b-9ed9-be100263de25)
